### PR TITLE
Update CI scripts to use inspektor 0.4.5

### PR DESCRIPTION
**Pull request by lmr**
Update avocado's CI scripts to use inspektor 0.4.5.

### CODE REVIEW

File: selftests/checkall [hunk 1]
```diff
@@ -80,12 +80,12 @@ results_dir_content() {
 
 [ "$SKIP_RESULTSDIR_CHECK" ] || RESULTS_DIR_CONTENT="$(ls $RESULTS_DIR 2> /dev/null)"
 if [ "$TRAVIS" == "true" ]; then
-    run_rc lint 'RESULT=$(mktemp); inspekt --exclude=.git lint &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
+    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git,.venv &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
```
Comment (ldoktor) [inline selftests/checkall]: would you please separate the `venv` change? It's really unrelated ti tgus cgabge,,,

File: selftests/unit/test_data_structures.py [hunk 2]
```diff
@@ -34,8 +34,7 @@ def test_compare_matrices(self):
         matrix1 = [["header", 51.7, 60], [1, 0, 0]]
         matrix2 = [["header", 57.2, 54], [2, 51, 0]]
         self.assertEqual(data_structures.compare_matrices(matrix1, matrix2),
-                         ([["header", '+10.6383', -10.0], ['+100',
-                          'error_51/0', '.']], 3, 1, 5))
+                         ([["header", '+10.6383', -10.0], ['+100.0', 'error_51/0', '.']], 3, 1, 5))
```
Comment (ldoktor) [inline selftests/unit/test_data_structures.py]: would you please split the line to not exceed 80 chars?

File: selftests/checkall (hunk 1 above)
Comment (lmr) [inline selftests/checkall]: OK

File: selftests/unit/test_data_structures.py (hunk 2 above)
Comment (lmr) [inline selftests/unit/test_data_structures.py]: OK

File: selftests/unit/test_xunit.py [hunk 3]
```diff
@@ -48,7 +48,7 @@ def test(self):
         self.test1._Test__status = 'PASS'
         self.test1.time_elapsed = 1.23
         self.junit = os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                     os.path.pardir, ".data", 'junit-4.xsd'))
+                                                  os.path.pardir, ".data", 'junit-4.xsd'))
```
Comment (ldoktor) [inline selftests/unit/test_xunit.py]: the same here, you could probably assign the `rel_path` first and then use `abspath` on that variable...

File: selftests/functional/test_basic.py [hunk 4]
```diff
@@ -1141,7 +1141,7 @@ class PluginsXunitTest(AbsPluginsTest, unittest.TestCase):
     def setUp(self):
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         self.junit = os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                     os.path.pardir, ".data", 'junit-4.xsd'))
+                                                  os.path.pardir, ".data", 'junit-4.xsd'))
```
Comment (ldoktor) [inline selftests/functional/test_basic.py]: This also exceeds the 80 chars. Would you please assign to a variable, or break the line?

File: selftests/unit/test_xunit.py (hunk 3 above)
Comment (lmr) [inline selftests/unit/test_xunit.py]: Sure

**Comment by lmr** [comment]
@ldoktor addressed all your comments, now just need to wait for Travis to finish running the tests.

**Comment by lmr** [comment]
Oh great, the compare matrices test failed. I suspect it's not related to my pull request, let's see...

File: selftests/unit/test_data_structures.py [hunk 5]
```diff
@@ -34,8 +34,8 @@ def test_compare_matrices(self):
         matrix1 = [["header", 51.7, 60], [1, 0, 0]]
         matrix2 = [["header", 57.2, 54], [2, 51, 0]]
         self.assertEqual(data_structures.compare_matrices(matrix1, matrix2),
-                         ([["header", '+10.6383', -10.0], ['+100',
-                          'error_51/0', '.']], 3, 1, 5))
+                         ([["header", '+10.6383', -10.0],
+                           ['+100.0', 'error_51/0', '.']], 3, 1, 5))
```
Comment (lmr) [inline selftests/unit/test_data_structures.py]: Ah, OK, found it

File: selftests/checkall [hunk 6]
```diff
@@ -80,12 +80,12 @@ results_dir_content() {
 
 [ "$SKIP_RESULTSDIR_CHECK" ] || RESULTS_DIR_CONTENT="$(ls $RESULTS_DIR 2> /dev/null)"
 if [ "$TRAVIS" == "true" ]; then
-    run_rc lint 'RESULT=$(mktemp); inspekt --exclude=.git lint &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
+    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
```
Comment (ldoktor) [inline selftests/checkall]: To those still actively using projects with old syntax and new syntax, this is a very ugly modification to the new inspector to be able to run them without any issues:

```diff
diff --git a/inspektor/cli/app.py b/inspektor/cli/app.py
index bdfc939..0238f04 100644
--- a/inspektor/cli/app.py
+++ b/inspektor/cli/app.py
@@ -17,10 +17,18 @@ Implements the base inspektor application.
 """
 import sys
 
+i = 0
+for i, arg in enumerate(sys.argv[1:]):
+    if arg.startswith("--"):
+        continue
+    break
+if i:
+    sys.argv = [sys.argv[0]] + sys.argv[i+1:] + sys.argv[1:i+1]
+print " ".join(sys.argv)
+
 from cliff.app import App
 from cliff.commandmanager import CommandManager
 
-
 class InspektorApp(App):
 
     def __init__(self):
```

**Pull request merged**